knowledge-base/badge-container-no-children.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Add a Badge to a Container that Cannot Have Children
+description: How to add a Telerik Badge to a container that cannot have children.
+type: how-to
+page_title: Add a Badge to a Container that Cannot Have Children
+slug: kb-badge-container-no-children
+tags: telerik, badge, avatar, container
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>
+                Badge for Blazor
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+I want to add the Telerik Badge to the [Telerik Avatar component]({%slug avatar-overview%}) or another container that cannot have child elements.
+
+## Solution
+
+To add the Telerik Badge for Blazor to the Avatar component or any other container that cannot have children, you need to wrap both the Badge and the container in an HTML element with the `position: relative`, `overflow: visible;`, and `display: inline-block;` CSS styles.
```

Comment:
The full component name makes the sentence clumsy.

`visible` is the default `overflow` style value. Instead of setting it explicitly, we can just suggest the developer to make sure the overflow value is not different.

```suggestion
To add the Badge to the Avatar component or any other container that cannot have children, you need to wrap both the Badge and the container in an HTML element with the `position: relative`, `overflow: visible;`, and `display: inline-block;` CSS styles.
```